src/components/select/examples/select-secondary-text.tsx

```diff
@@ -1,4 +1,4 @@
-import { Option } from '@limetech/lime-elements';
+import { LimelSelectCustomEvent, Option } from '@limetech/lime-elements';
```

Comment:
It looks like this import has been missing since 2024-02-02. Without it I can't run `npm start` without getting a build error.

I guess we haven't been working very actively in lime-elements since January…? 🤷‍♂️

Reply:
January? :/ 
but this new example got just merged. 
However, it's strange that it could build then.